src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+
+    private static Random randomColor = new Random();
+
     public String getRandomColor() {
-        return null;
+        Colors[] colors = Colors.values();
+        int randomIndex = randomColor.nextInt(colors.length);
+        return String.valueOf(colors[randomIndex]);
```

Comment:
Instead of using `String.valueOf()`, which internally calls `toString()`, use the `name()` method to get the name of the enum constant. This ensures that you get the exact name of the enum constant even if `toString()` is overridden in the enum.